### PR TITLE
Updated index.html to add a button 'Edit Text' which opens a modal fo…

### PR DESCRIPTION
**Pull request by asingh217**
Updated index.html to add a button 'Edit Text' which opens a modal dialog for user to edit the input.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,16 +30,43 @@ <h1 style="color: #49a1e0"> <img src="static/shabdawali_logo.png" height="50px"
...
                             <p>
-                                <input type="checkbox" id="check-blinkCursor" name="check-blinkCursor" checked>
-                                <label for="check-blinkCursor">Display Blinking Cursor.</label>
-
-                                <input type="checkbox" id="check-keySound" name="check-keySound">
-                                <label for="check-keySound">Play keypress sound.</label>
-
-                                <input type="checkbox" id="check-dynamicPause" name="check-dynamicPause" checked>
-                                <label for="check-dynamicPause">Dynamic pauses</label>
+                                <table width="100%"> <!--Added Table to aligning the checkboxes and button-->
```
Comment (amitguptagwl) [inline index.html]: As we're using the Bootstrap CSS framework, you can achieve it without the table tag but `<div class="row">..`

**Comment by amitguptagwl** [comment]
Thanks for your reviews @ashishpatelcs

File: index.html (hunk 1 above)
Comment (asingh217) [reply index.html]: @ashishpatelcs  suggested to leave the checkboxes as is. I used the table tag only to align the newly added button 'Edit Text'. If this is part of a separate PR then i will remove the table tags completely.

Comment (ashishpatelcs) [reply index.html]: @asingh217 Yes you can remove the table tags. The buttons can be aligned using a div, if necessary.

**Comment by asingh217** [comment]
Updated and committed. Please review.

**Comment by asingh217** [comment]
Great!!

**Comment by amitguptagwl** [comment]
@asingh217 Unfortunately I've merged another PR. Hence you can see the conflicts. You'll have to resolve them first so that this PR can be merged.

**Comment by asingh217** [comment]
@amitguptagwl I don't see any conflict yet. I already rebased for PR#23. Can you share the PR number which you merged causing conflicts.
In git it says "This branch has no conflicts with the base branch"

**Comment by amitguptagwl** [comment]
I believe it was my bad. Thanks for your PR and congratulations. If you want to support more opensource projects. You may join [Team](https://github.com/socialat/team/issues)

**Comment by asingh217** [comment]
No problem. Thanks for the support 👍

**Pull request merged**